logstash-core/lib/logstash/runner.rb

```diff
@@ -285,6 +285,12 @@ def execute
     require "logstash/util/java_version"
     require "stud/task"
 
+    if Process.euid() == 0
+      run_as_root = setting("run_as.root") == true
+      raise(RuntimeError,"can not run logstash as root") if !run_as_root
```

Comment:
```suggestion
      raise(RuntimeError,"Cannot run logstash as root") if !run_as_root
```

Reply:
Or perhaps "Logstash cannot be run as root"

Reply:
This suggestion LGTM!